matchms/importing/load_from_usi.py

```diff
@@ -0,0 +1,42 @@
+import json
+import numpy as np
+import requests
```

Comment:
Requests got installed as a dependency of a dependency. When you import it directly it should also be a direct dependency of matchms.

Can you add requests to he conda config files in `conda/` directory. Where ever you see `pyteomis` add `requests`?